apis/beacon/states/committee.yaml

```diff
@@ -0,0 +1,43 @@
+get:
+  operationId: "getEpochCommittees"
+  summary: "Get all committees for epoch"
+  description: "Retrieves the committees for the given state at the given epoch."
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+    - name: epoch
+      description: "Epoch for which to calculate committees"
+      in: path
+      required: true
+      schema:
+        $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+    - name: index
+      description: "Committee index"
+      in: query
+      required: false
+      schema:
+        $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+    - name: slot
```

Comment:
Must the `slot` be from the correct `epoch`?
If so, is there a good way to note those? If not, can we note how these interact (e.g. pull all from epoch, then filter by slot -- so if slot is not within epoch, will return empty)

Reply:
we cannot specify min/max to depend on another param. I think is fine to return empty array along with description that is expected behavior

Reply:
Can we state that we return a 400 if the slot is not in the epoch?